decidim-core/spec/system/user_profile_spec.rb

```diff
@@ -48,6 +48,19 @@
       expect(page.find('meta[name="robots"]', visible: false)[:content]).to eq("noindex")
     end
 
+    it "displays the correct links for profile activity" do
+      within "#filters" do
+        expect(page).to have_link("All activity types", href: decidim.profile_activity_path(nickname: user.nickname, filter: { resource_type: "all" }))
+        expect(page).to have_link("Collaborative draft", href: decidim.profile_activity_path(nickname: user.nickname, filter: { resource_type: "Decidim::Proposals::CollaborativeDraft" }))
+        expect(page).to have_link("Proposal", href: decidim.profile_activity_path(nickname: user.nickname, filter: { resource_type: "Decidim::Proposals::Proposal" }))
+        expect(page).to have_link("Comment", href: decidim.profile_activity_path(nickname: user.nickname, filter: { resource_type: "Decidim::Comments::Comment" }))
```

Comment:
Can you add an example with actually clicking and having the resource (for instance a Comment made by the participant) 🙏🏽